response_operations_ui/common/remove_whitespace_from_survey.py

```diff
@@ -0,0 +1,2 @@
+def remove_whitespace_from_survey_name(survey):
```

Comment:
is there no utils file you could put this in?

Reply:
This does not remove whitespace , it removes spaces . To remove whitespace   my_var=''.join(my_var.split()).
I am not sure if this needs to be a separate function . If it is then the name should not include 'survey_name' as it will work with any string